Yoast/wordpress-seo
# Restore title separators focus style

## Summary

This PR can be summarized in the following changelog entry.

* Restore focus indication on the title separators user interface

## Relevant technical choices:
- removes the box-shadow, as the current style can be done with border only
- added bonus: border works also in Windows High Contrast mode
- on focus, the border style changes to `dashed`

Screenshot of normal state:

<img width="684" alt="normal state" src="https://user-images.githubusercontent.com/1682452/52128488-628c6100-2635-11e9-8bfa-d02b17e3d339.png">

Screenshot of focus state:

<img width="690" alt="screenshot 2019-02-06 at 17 09 37" src="https://user-images.githubusercontent.com/1682452/52372707-256f0700-2a59-11e9-93d3-0f98d8d3a9b7.png">

## Test instructions
- on trunk: 
  - use the keyboard and focus the first separator
  - notice there's no style change
  - use arrow keys to focus the other separators: still no style change
- on this branch:
  - build the CSS
  - repeat the steps above
  - notice the style change

Update: test also in the Configuration wizard:
- yarn link the yoast-components branch `12144-title-separator-focus`
- build the assets
- go in the Configuration wizard > step 9
- check the title separators look the same as in the Settings page and the focus style works the same

## UI changes

Fixes #12144
